source/game/system/map/MapdataCannonPoint.hh

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
+
+#include "game/kart/KartMove.hh"
+
+#include <egg/math/Vector.hh>
+
+namespace System {
+
+class MapdataCannonPoint {
+public:
+    struct SData {
+        EGG::Vector3f pos;
+        EGG::Vector3f rot;
+        u16 id;
+        s16 propertyIdx;
+    };
+
+    MapdataCannonPoint(const SData *data);
+    void read(EGG::Stream &stream);
+
+    /// @beginGetters
+    const EGG::Vector3f &pos() const;
+    const EGG::Vector3f &rot() const;
+    u16 id() const;
+    s16 parameterIdx() const;
+    /// @endGetters
+
+private:
+    const SData *m_rawData;
+    EGG::Vector3f m_pos;
+    EGG::Vector3f m_rot;
+    u16 m_id;
+    s16 m_parameterIdx; ///< Index into the table of cannon properties. Used to determine speed,
+                       ///< height, and decel
```

Comment:
```suggestion
    s16 m_parameterIdx; ///< Index into the table of cannon properties. Used to determine speed,
                       ///< height, and decel
```
Indentation seems wrong here. Can you run the linter again on the source files changed in this PR?